CONTRIBUTING.md

```diff
@@ -17,29 +17,34 @@ First, checkout the main [Moloch README](README.md) for information on how to bu
 
 **Then, get some test data!**
 
-* Make sure node 10 is in your path
+* Make sure `node` is in your path
 * Install and Start Elasticsearch
 * Use `easybutton-build.sh` or `configure` and `make` to build everything
 * Run `make check` from the top level directory, this will
   * run `npm ci` everywhere
-  * run `tests.pl` and `tests.pl --viewer` in the tests directory
+  * run `tests.pl` and `tests.pl --viewer` in the tests directory (this loads PCAPs)
 
 > **Note:** this will only work if viewer is not already running.
 
 You should now have test data loaded, so let's **start the web app**:
 
-* Move to the Moloch viewer directory
+* Move to the viewer directory
 * Run `npm ci`
```

Comment:
do you still need this in the viewer directory?

Reply:
Yeah, there are still a few node modules in the viewer directory. I opted to leave them here because they are specific to viewer (no overlap with any of the other projects). But, we could take this even further and put everything at the top level. I thought that was unnecessary since we need a package.json for each file anyway.